includes/firewall-limits.md

```diff
@@ -13,11 +13,12 @@
 | Resource | Limit |
 | --- | --- |
 | Data throughput |30 Gbps|
-|Rule limits|10,000 unique source/destinations in network rules|
+|Network Rule limits|10,000 unique source/destinations in network rules|
+|Application Rule limits|no limit|
```

Comment:
```suggestion
```

Reply:
@MaxMelcher 
According to Firewall PM, this is the correct statement.

Reply:
@vhorne I am fine with both - please merge it.